### PR TITLE
Add trusted profiles list operations: claimrule and link

### PR DESCRIPTION
**Pull request by pooraniss-ibm**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

**Comment by pooraniss-ibm** [comment]
Screenshot 2021-11-02 at 15 20 01
Screenshot 2021-11-02 at 15 59 46
Screenshot 2021-11-02 at 16 05 28

File: website/docs/d/iam_trusted_profile_link_list.html.markdown [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+data "ibm_iam_trusted_profile_link_list" "iam_trusted_profile_link_list" {
+	profile_id = "profile_id"
+}
+```
+
+## Argument Reference
+
+Review the argument reference that you can specify for your data source.
+
+* `profile_id` - (Required, Forces new resource, String) ID of the trusted profile.
```
Comment (kavya498) [inline website/docs/d/iam_trusted_profile_link_list.html.markdown:25]: It is not a force new string
please update docs according to schema

File: website/docs/d/iam_trusted_profile_link_list.html.markdown [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+## Attribute Reference
+
+In addition to all argument references listed, you can access the following attribute references after your data source is created.
+
+* `id` - The unique identifier of the iam_trusted_profile_link_list.
+* `links` - (Required, List) List of links to a trusted profile.
+  Nested scheme for **links**:
+	* `id` - (String) the unique identifier of the claim rule.
```
Comment (kavya498) [inline website/docs/d/iam_trusted_profile_link_list.html.markdown:34]: Please re-arrage attributes in alphabetical order..
this is applicable to all other docs as well

**Comment by kavya498** [comment]
@pooraniss-ibm ,
please resolve all conflicts
currently there are 8 commits in this Pr.. Please reduce it to one or two

**Comment by kavya498** [comment]
https://github.com/IBM-Cloud/terraform-provider-ibm/issues/3200

**Comment by kavya498** [comment]
Closing this PR, duplicate of [3352](https://github.com/IBM-Cloud/terraform-provider-ibm/pull/3352)

**Pull request closed without merging**